frontend/src/components/TypicalOverlayScrollbar/typicalOverlayScrollbar.scss

```diff
@@ -0,0 +1,3 @@
+.overlay-scrollbar {
+	height: calc(100% - 70px);
```

Comment:
??

Reply:
Sorry, my bad. It should be `height: 100%`, fixed it :slightly_smiling_face: :+1: